yoti/templates/yoti-button.html.twig

```diff
@@ -6,6 +6,7 @@
  * Available variables:
  * - button_id: Yoti Button ID
  * - scenario_id: Yoti Scenario ID
+ * - sdk_id: Yoti SDK ID
```

Comment:
Are we just calling it `Yoti SDK ID` here, and omitting `client`, since it's obvious it's the client? I was thinking it might be more consistent if we call it `Yoti Client SDK ID`/`Client SDK ID` everywhere

Reply:
I was just mirroring the Drupal config key, which is currently `yoti_sdk_id` - I'll update the comment here

Reply:
@echarrod changed `SDK ID` to `Client SDK ID` in 67367b6

Reply:
As discussed, reinstated `getAppID()` in 3c3651b - this was removed in #48 (that this PR corrects) - we will remove this getter when we remove from config and admin form.